### ISSUE TITLE
Allow ‘Previous Version’ history of products

### ISSUE THREAD

**Issue by cklosowski** [easydigitaldownloads/easy-digital-downloads#7791]
### Explain your enhancement (please be detailed)
It would be helpful in some store’s cases to have a history of different versions of files that are being sold. While it is possible to just keep adding new files to the product with the versions in the name, that would show to all areas where the download list is used , which might not be ideal.

### Justification or use case
After reading an email from @polevaultweb, I’m envisioning a separate meta box or file list that is ‘past versions’ where store owners can ‘transfer’ a version to that list, while maintaining their current versions as the ‘product files’.

With a separate list we can avoid having all past versions show up in purchase receipts, and just have them show in the short codes for account management.

Before we start work on this, I’d like to get some comps of what it might look like on the front end, as well as what a new meta box/file download management interface looks like as well. Some questions:

1) Do we make the store owner manually move a file to the “Previous Versions”
2) Do we have a quick button to ‘move to past versions list’
3) This should still work with SL as that has a drop down of which file to deliver for updates, meaning SL can deliver the latest only.

**Comment by ashleyfae** [comment]
> Do we have a quick button to ‘move to past versions list’

I'm a fan of whatever makes this flow quicker/easier.

Also support with Git updater. :heart:

**Comment by brashrebel** [comment]
I'm very curious: Is this applicable to many non-software sellers? I know it makes sense for them but what about other product types? Essentially what is the argument for putting this in core vs. in SL?

**Comment by polevaultweb** [comment]
@brashrebel you're right, this probably should be a SL feature - but it would poss change the inner workings of the download files metabox.

**Comment by emprer** [comment]
Did we go anywhere with this? I need this feature on my store and I've been struggling to find a solution.

**Comment by emprer** [comment]
Did anything happen with this? I need to implement this feature on my website.

**Comment by grand-lotus-iroh** [comment]
> Did anything happen with this? I need to implement this feature on my website.

Lazy developers but [this might be a proper solution](https://www.pluginsandsnippets.com/downloads/edd-product-versions/).
